### ISSUE TITLE
vbguest  "installation was found but no tools to rebuild or start them"

### ISSUE THREAD

**Issue by iainhouston** [geerlingguy/drupal-vm#1995]
### Issue Type

 - Bug Report / Support Request
### Your Environment

```
PHP 7.4.0 (cli) (built: Nov 29 2019 16:18:44) ( NTS )
Copyright (c) The PHP Group
Zend Engine v3.4.0, Copyright (c) Zend Technologies
    with Zend OPcache v7.4.0, Copyright (c), by Zend Technologies
Composer version 1.9.1 2019-11-01 17:20:17
Vagrant 2.2.6
VirtualBox 6.0.14r133895
ansible 2.9.1
  config file = None
  configured module search path = ['/Users/iainhouston/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/Cellar/ansible/2.9.1/libexec/lib/python3.7/site-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 3.7.5 (default, Nov 30 2019, 08:22:32) [Clang 11.0.0 (clang-1100.0.33.12)]
NodeJS Version v13.2.0
npm Version 6.13.1
vagrant-auto_network (1.0.3, global)
  - Version Constraint: > 0
vagrant-hostsupdater (1.1.1.160, global)
  - Version Constraint: > 0
vagrant-vbguest (0.22.0, global)
  - Version Constraint: > 0
Developer Edition of Mozilla Firefox 71.0b12
```

### Your OS

 - **macOS** (Catalina)

### Full console output

[Console output gist](https://gist.github.com/iainhouston/e5a484e81febbcbecca7a282407ac3c0)

### Summary

vbguest builds correctly when the box  is first provisioned but has to rebuild every `vagrant up` or `vagrant reload`

**Comment by iainhouston** [comment]
This happens with both `vagrant_box: geerlingguy/ubuntu1804` and `vagrant_box: geerlingguy/drupal-vm`

It does rather slow things down: having to rebuild each `vagrant up`

Should have added above that : 
```
% ruby --version
ruby 2.6.3p62 (2019-04-16 revision 67580) [universal.x86_64-darwin19]
```

although probably irrelevant as this is seemingly a guest OS issue

**Comment by iainhouston** [comment]
This is an issue with `vbguest` reported at their GitHub page https://github.com/dotless-de/vagrant-vbguest/issues/362

**Comment by phizev** [comment]
Just as a note to anyone else whom may have come across this, this has now been fixed in `vagrant-vbguest 0.22.1`, just run `vagrant plugin update` to fix it.
